<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Timer</class>
 <widget class="QWidget" name="Timer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>225</width>
    <height>157</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>150</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>GroupBox</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Time On</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QTimeEdit" name="onTmeSpinBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Time Off</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QTimeEdit" name="offTmeSpinBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="updateBtn">
          <property name="text">
           <string>update times</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="revertBtn">
          <property name="text">
           <string>revert times</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
